src/desktop/components/auth_modal/templates/_gdpr_signup.jade

```diff
@@ -0,0 +1,88 @@
+.gdpr-signup
+  form.gdpr-signup__form(
+    action=sd.AP.loginPagePath + "?redirect-to=" + redirectTo
+    method='POST'
```

Comment:
The key pieces: adding `=` on line 3 (thanks @kanaabe and @xtina-starr ) and putting `POST` in quotation marks.